# BirdCollection.srv
task_2s/Bird[] birds
---